src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final int SALARY_SPLIT_INDEX = 2;
+    private static final int NAME_SPLIT_INDEX = 1;
+    private static final int DATE_SPLIT_INDEX = 0;
+    private SimpleDateFormat sdformat = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
If you create a formatter, make it a constant field. Since formatters are typically constant values, they should be static final fields with uppercase naming. This will make your code easier to understand and maintain.

Reply:
please use proper naming + make it a constant